wled00/bus_manager.h

```diff
@@ -575,6 +559,18 @@ class BusManager {
     }
   }
 
+  void setSegmentCCT(int16_t cct, bool allowWBCorrection = false) {
```

Comment:
You are hijacking bus manager?

Reply:
Would not call it hijacking :) more setting a value that applies globally for all busses. Essentially it is a `BusManager::setPixelColor()`, just separately for just CCT. The alternatives would be 1) passing it as part of `setPixelColor()` like you implemented initially, which I liked, but increased complexity (two `setPixelColor()` methods) and 2) having it as a global var that can be modified from all parts of WLED, running the risk to unintentionally setting an invalid value since all the checks run by `setSegmentCCT` would need to be implemented in `strip.service()` which is less clean imo.

Reply:
Ah! Ok, I thought it was linked to segment as seen in strip.